DEVELOPMENT.rst

```diff
@@ -16,14 +16,18 @@ a. Check requirements
 '''''''''''''''''''''
 
 Make sure that you have installed `Python (>= 3.4) <https://www.python.org/>`_,
-`Node.js (>=4.x) <https://nodejs.org/>`_, `npm - Node Package Manager (>=3.x)
-<https://npmjs.org/>`_ and `Git <http://git-scm.com/>`_ on your system. You also
-need build-essential packages and header files and a static library for Python.
+`Node.js (>=4.x) <https://nodejs.org/>`_, `Yarn <https://yarnpkg.com/>`_ and
```

Comment:
Is it necessary that Node/npm is installed parallel to yarn? Did we need node anymore? I didn't test this but yarn should be a drop in replacement for npm. Also in the yarn installation guide it does not tell to install node.